src/components/Section/Legal/Investigations/HistoryItem.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import HistoryItem from './HistoryItem'
+
+describe('The legal investigations history item component', () => {
+  it('can select values', () => {
+    let updates = 0
+    const onUpdate = () => { updates++ }
+    const component = mount(<HistoryItem onUpdate={onUpdate} />)
+    component.find('.legal-investigations-history-agency .investigative-agency-dod input').simulate('change')
+    component.find('.legal-investigations-history-completed .year input').simulate('change')
+    component.find('.legal-investigations-history-issued input').simulate('change')
+    component.find('.legal-investigations-history-granted .year input').simulate('change')
+    component.find('.legal-investigations-history-clearance .clearance-level-none input').simulate('change')
+    expect(updates).toBe(5)
```

Comment:
The not-applicables (agency, completed, granted, and clearance) are missing change tests.